lib/tasks/db.rake

```diff
@@ -0,0 +1,204 @@
+    db = Mongo::Connection.new.db(db_name)
+
+    CommentThread.create_indexes
+    Comment.create_indexes
+    Content.delete_all
+    coll = db.collection('contents')
+    args[:num].to_i.times do
+      doc = {'_type' => 'CommentThread',
+             'anonymous' => [true, false].sample,
+             'at_position_list' => [],
+             'tags_array' => [],
+             'comment_count' => 0,
+             'title' => Faker::Lorem.sentence(6),
+             'author_id' => rand(1..10).to_s,
+             'body' => Faker::Lorem.paragraphs.join('\n\n'),
+             'course_id' => COURSE_ID,
+             'created_at' => Time.now,
+             'commentable_id' => COURSE_ID,
+             'closed' => [true, false].sample,
+             'updated_at' => Time.now,
+             'last_activity_at' => Time.now,
+             'votes' => {
+                 'count' => 0,
+                 'down' => [],
+                 'down_count' => 0,
+                 'point' => 0,
+                 'up' => [],
+                 'up_count' => []
+             }
+      }
+      coll.insert(doc)
+    end
+    Tire.index('comment_threads').delete
+    CommentThread.create_elasticsearch_index
+    Tire.index('comment_threads') { import CommentThread.all }
+  end
+
+  task :seed_fast => :environment do
+    ADDITIONAL_COMMENTS_PER_THREAD = 20
+
+    config = YAML.load_file('config/mongoid.yml')[Sinatra::Base.environment]['clients']['default']
+    connnection = Mongo::Connection.new(config['hosts'][0].split(':')[0], config['hosts'][0].split(':')[1])
+    db = Mongo::Connection.new.db(config['database'])
+    coll = db.collection('contents')
+    Comment.delete_all
+    CommentThread.each do |thread|
+      ADDITIONAL_COMMENTS_PER_THREAD.times do
+        doc = {
+            '_type' => 'Comment',
+            'anonymous' => false,
+            'at_position_list' => [],
+            'author_id' => rand(1..10).to_s,
+            'body' => Faker::Lorem.paragraphs.join('\n\n'),
+            'comment_thread_id' => BSON::ObjectId.from_string(thread.id.to_s),
+            'course_id' => COURSE_ID,
+            'created_at' => Time.now,
+            'endorsed' => [true, false].sample,
+            'parent_ids' => [],
+            'updated_at' => Time.now,
+            'votes' => {
+                'count' => 0,
+                'down' => [],
+                'down_count' => 0,
+                'point' => 0,
+                'up' => [],
+                'up_count' => []
+            }
+        }
+        coll.insert(doc)
+      end
+    end
+  end
+
+  task :seed => :environment do
+    include FactoryGirl::Syntax::Methods
+    FactoryGirl.find_definitions
+
+    Comment.delete_all
+    CommentThread.delete_all
+    User.delete_all
+    Notification.delete_all
+    Subscription.delete_all
+    Tire.index 'comment_threads' do
+      delete
+    end
+    CommentThread.create_elasticsearch_index
+
+    beginning_time = Time.now
+
+    create_list(:user, 10)
```

Comment:
This looks like the test users will be slightly different than create_test_user.  Will they have the same username?  Does that matter?